### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by saraedum**
tweak zenodo integration and call it e-antic (instead of E-ANTIC) consistently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
-# E-ANTIC — (Real Embedded) Algebraic Number Theory
+<!--
+<p align="center">
+    <img alt="logo" src="https://github.com/flatsurf/e-antic/raw/master/logo.svg?sanitize=true" width="300px">
```
Comment (saraedum) [inline README.md:3]: There's no logo yet. @alejandrarobles is working on ants carrying a root sign but the README already looks quite reasonable without it.

**Comment by saraedum** [comment]
@videlec please merge if this looks good to you.

File: README.md [hunk 2]
```diff
@@ -70,28 +84,48 @@ For more detailed but generic instructions please refer to the INSTALL file.
...
-To build all of e-antic package, you need a fairly recent C++ compiler and
-probably some packages that might not be readily available on your system. If
-you don't want to use your distribution's packages, you can provide these
-dependencies with Conda. Download and install
-[Miniconda](https://conda.io/miniconda.html), then run
+To build all of e-antic, you need a fairly recent C++ compiler and probably
+some packages that might not be readily available on your system. If you don't
+want to use your distribution's packages, you can use these dependencies from
+[conda-forge](https://conda-forge.org). Download and install
+[Mambaforge](https://github.com/conda-forge/miniforge#mambaforgeg), then run
```
Comment (slel) [inline README.md:101]:
```diff
- https://github.com/conda-forge/miniforge#mambaforgeg
+ https://github.com/conda-forge/miniforge#mambaforge
```

File: README.md [hunk 3]
```diff
@@ -70,28 +84,48 @@ For more detailed but generic instructions please refer to the INSTALL file.
...
     make
-    make check
-    make html # to build the documentation
+    make check  # to run our test suite
+    make html  # to build the documentation
+
+## How to Cite this Project
+
+If you have used this project in the preparation of a publication, please cite
+it as described [on our zenodo site](https://doi.org/10.5281/zenodo.5166953).
```
Comment (slel) [inline README.md:121]: zenodo site -> zenodo page?

**Pull request merged**